src/vscode/launch-config.ts

```diff
@@ -130,9 +170,59 @@ export class VsCodeLaunchConfig extends Component {
   public addConfiguration(cfg: VsCodeLaunchConfigurationEntry) {
     this.content.configurations.push(cfg);
   }
+
+  /**
+   * Adds an input variable with type `promptString` to `.vscode/launch.json`.
+   *
+   * See https://code.visualstudio.com/docs/editor/variables-reference#_input-variables for details.
+   * @param cfg VsCodeLaunchPromptStringInputEntry
+   */
+  public addPromptStringInput(cfg: VsCodeLaunchPromptStringInputEntry) {
+    this.content.inputs.push({
```

Comment:
Something like this should work:
```suggestion
    this.content.inputs ??= [];
    this.content.inputs.push({
```